var/spack/repos/builtin/packages/relion/package.py

```diff
@@ -40,15 +39,19 @@ class Relion(CMakePackage, CudaPackage):
             description='The build type to build',
             values=('Debug', 'Release', 'RelWithDebInfo',
                     'Profiling', 'Benchmarking'))
+    variant('mklfft', default=True, description='Use MKL rather than FFTW for FFT')
+    variant('allow_ctf_in_sagd', default=True, description='Allow CTF-modulation in SAGD, as specified in Claim 1 of patent US10,282,513B2')
 
     depends_on('mpi')
     depends_on('cmake@3:', type='build')
-    depends_on('fftw precision=float,double')
+    depends_on('fftw precision=float,double', when='~mklfft')
     depends_on('fltk', when='+gui')
     depends_on('libtiff')
 
     depends_on('cuda', when='+cuda')
     depends_on('cuda@9:', when='@3: +cuda')
+    depends_on('intel-tbb', when='~cuda')
+    depends_on('intel-mkl', when='~cuda +mklfft')
```

Comment:
```suggestion
    depends_on('mkl', when='~cuda +mklfft')
```
`mkl` is a virtual package provided by both `intel-mkl` and `intel-parallel-studio+mkl`

Reply:
Good point. Will update for both mkl and tbb virtual packages.